include/RevRegFile.h

```diff
@@ -32,11 +32,12 @@ namespace SST::RevCPU {
 struct RevInst;
 
 /// BoxNaN: Store a boxed float inside a double
-inline void BoxNaN( double* dest, const void* value ) {
+inline void BoxNaN( double* dest, const float* value ) {
   uint32_t i32;
-  memcpy( &i32, value, sizeof( float ) );                 // The FP32 value
+  memcpy( &i32, value, sizeof( i32 ) );                   // The FP32 value
```

Comment:
What's the difference here? Wouldn't it be the same?

Reply:
Just a good programming practice; when copying into a variable with `memcpy()`, it is good to make the size of the copy equal to the size of the variable being copied into.

Reply:
As I recall, there is a `static_assert` making sure that the sizes of `float` and `i32` are the same.